platforms/documentation/docs/src/snippets/dependencyManagement/customizingResolution-conditionalSubstitutionRule/groovy/buildSrc/src/main/groovy/myproject.java-library-conventions.gradle

```diff
@@ -3,5 +3,5 @@ plugins {
 }
 
 repositories {
-    ivy { url "file://${rootProject.projectDir}/repo" }
+    ivy { url "${rootProject.projectDir.toURI().toASCIIString()}/repo" }
```

Comment:
Does this work?

url = uri(rootProject.layout.projectDirectory.dir("repo"))

Reply:
Yes. Changed.